app/routes/docs.notices.producers.mdx

```diff
@@ -49,6 +49,13 @@ also happy to work with the mission teams to help construct your alerts.
          Draft Your Schema
       </ProcessListHeading>
       As a GCN Notice producer, you can create your own instrument-specific schema. Please contribute your schema to our [GitHub repository](https://github.com/nasa-gcn/gcn-schema), placing it in a folder under <code>gcn/notices/<i>mission</i></code> and submit a pull request for the GCN Team to review. For details, please refer to the [schema documentation](/docs/notices/schema).
+
+      ### Tag a Schema Version
+      The schema follows the principles of [Semantic Versioning](https://semver.org). Updates are categorized as major, minor, or patch, depending on the type of change. A major change is defined as any modification that breaks compatibility with the core schema, such as a change in a property or the addition of a new property that affects the mission pipelines. Introducing a new mission and new core schema constitutes a minor change, while modifying the new mission schema and adding a new property to the core schema are considered patches.
```

Comment:
Need to make it clear that tagging schema versions is done by the GCN team, not the mission teams.

Reply:
Since these are instructions for mission points of contact, and they do not and cannot tag versions themselves, isn't it a distraction to put these instructions here in the first place?

Reply:
We've talked about including instructions for GCN team processes in these documents, but it needs to be clear where the responsibility lies.

Reply:
Yes, it's for us. However we can start with the statement stating, GCN Team is responsible for version update based on type of modification in core schema. It's good for producer to know the process of version updates.

Reply:
Given that these are instructions for maintainers, not notice producers, I think it would be better if these instructions were elsewhere: for example, in the README for the package.

If you like, you could add a single sentence here, to the effect of: "When you are ready to start publishing your alerts, contact the GCN maintainers, who will tag a new version of the schema. (For maintainers, see instructions for doing a release.)"

Reply:
Sounds equally good.

Reply:
OK, what are we going to do about this now?

Reply:
@Vidushi-GitHub and I discussed.  This is going to move to the README.